\documentclass[11pt,letterpaper]{article}

\usepackage{graphicx}

\usepackage{multicol}
\usepackage[margin=15mm]{geometry}

\usepackage{sidecap}
\sidecaptionvpos{figure}{c}

\usepackage[super,sort&compress]{natbib}

% define multiple reference lists
\usepackage{multibib}
\newcites{ref,own}{References,Publication list}

% define new environments
\usepackage{amsthm}
\theoremstyle{definition}
\newtheorem*{hypothesis}{Hypothesis}
\newtheorem*{objective}{Objective}

% disable chapter, section, and subsection numbering
\setcounter{secnumdepth}{-1}

\begin{document}

\input{./title.tex}

\section{Overview}

Medulloblastoma is the most comon solid childhood malignancy \citeref{mainprize00}. Current therapy for medulloblastoma --- including surgical resection, radiation of the entire brain and spinal cord, and aggressive chemotherapy --- yields five-year survival rates of 60-70\% \citeref{gajjar06}. Survivors are often left with significant neurological, intellectural, and physical disabilities secondary to the effects of these non-specific, cytotoxic therapies on the developing nervous system \citeref{spiegler04,mabbott05}.
Recent evidence suggests that medulloblastoma in fact comprises a group of molecularly and clinically distinct entities whose clinical and genetic differences may require separate therapeutic strategies \citeref{thompson06,kool08,northcott11a,remke11,cho11}. Four principal subgroups\citeref{taylor12} of medulloblastoma have been identified: WNT, SHH, Group 3, and Group 4, and there is preliminary evidence for clinically significant subdivisions of the subgroups \citeref{northcott11a,cho11,remke11,taylor12,northcott11b}. Targeted therapies based on the genetics of the disease are not currently in use. However, inhibitors of the Sonic Hedgehog (Shh) pathway activator, Smoothened, have shown some early evidence of efficacy \citeref{rudin09}. With a deeper understanding of the molecular biology of medulloblastoma subgroups, we hope to herald a new era of medullblastoma treatment based on selective, specific, targeted therapy.

At present, there are at least three main obstacles that hinders the development of targeted therapy against medulloblastoma molecular subgroups:

\begin{enumerate}
	\item The lack of a clinically applicable assay for molecular subgrouping of medulloblastoma.
	\item The paucity of actionable targets for WNT, Group 3, and Group 4 medulloblastomas.
	\item Clinical trials of targeted therapy proceed without confirmed identifications of molecular targets in recurrent mebulloblastomas.
\end{enumerate}

The objectives of my thesis are to provide viable solutions to these issues and to demonstrate the clinical significance of molecular subgrouping.

\subsection{Aim 1: Molecular classification of medublloblastoma in clinical contexts}

Although the retrospective classification of medulloblastoma has been scientifically informative, molecular subgrouping has not been applied in the context of a propsective clinical trial. One major obstacle is the lack of fresh-frozen samples for the most clinical cases. Expression profiling, on which molecular classification was based, depends on the availability of high-quality RNA. In contrast, clinical samples are routinely subjected to formalin-fixation and paraffin-embedding (FFPE), which preserves tissue integrity but causes nucleic acid degradation. To faciliate the development of therapy specfically targeted against molecular subgroups, we sought to establish an molecular subgrouping assay that can be clinically applied on FFPE samples. In collaboration with Paul Northcott, I established a simple analytic pipeline to molecular subgrouping using expression data generated by Nanostring assays, and demonstrated its high classification accuracy on FFPE samples \citeown{northcott12}.

\subsection{Aim 2: Target identification through copy-number profiling of medulloblastoma}

After having established a clinically applicable molecular classification methodology, I turned to the problem of identifying molecular targets in medulloblastoma. Unlike SHH medulloblastomas, actionable targets for WNT, Group 3, and Group 4 tumours have yet been identified. However, prior attempts may have been underpowered to discriminate the genomic differences among the four moleulcar subgroups. To this end, the Medulloblastoma Advanced Genomics International Consortium (MAGIC), consisting of scientists and physicians from 43 cities across the globe, has gathered >1200 medulloblastomas. Paul Northcott and I have analyzed the genomic copy-number profiles of the tumorus by Single Nucleotide Polymorphism (SNP) arrays. We have identified genes and pathways that characterize each medulloblastoma subgroup \citeown{shih12}.

\subsection{Aim 3: Demarcating the genomic differences between primary and recurrent tumours}

To date, most clinical efforts and changes in pediatric tumour treatment has been in the optimization of chemotherapy and radiation protocols \citeref{bouffet10}. While survival rates for medulloblastoma patients have improved over the years, patients with recurrent medulloblastoma currently have no effective treatment option. These patients are often enrolled in clinical trials based on diagnostic tests of their primary disease, under the likely erroneous assumption that recurrent tumours are identical to their primary counterparts. MAGIC has collected one of the largest set of matched primary-recurrent medulloblastomas. I will analyze these tumorus by copy-number profiling and Whole-Genome Sequencing (WGS). From the data, I will identify the genomic differences between primary and recurrence and establish their biological significance.


\section{Summary of progress}

Aim 1 has been completed\citeown{northcott12}, and Aim 2 is near completion\citeown{shih12}.
I have begun working on Aim 3. I am waiting for the generation of WGS data on the primary and recurrent samples in our collaborator's lab (Dr. Marco Marra, Michael Smith Genome Sciences Centre).
Aim 4 is pending the collection and submission of additional tumours samples for expression profiling.

\nociteown{wu12,rausch12,dubuc12}

\section{Aim 1: Molecular classification of medublloblastoma in clinical contexts}

\begin{objective}
To develop a clinically applicable assay for molecular classification of medulloblastoma.
\end{objective}

\section{Aim 2: Target identification through copy-number profiling of medulloblastoma}

\begin{hypothesis}
Each medulloblastoma molecular subgroup is characterized by specific genomic aberrations.
\end{hypothesis}

\subsection{Minor aim: Development of an improved tool for identifying recurrent somatic copy-number aberrations}

\begin{objective}
To develop a fast algorithm for population-based identification of recurrent somatic copy-number aberrations.
\end{objective}

\section{Aim 3: Demarcating the genomic differences between primary and recurrent tumours}

\begin{hypothesis}
Primary and recurrent medulloblastomas are genetically different, which may cause differential responses to therapy.
\end{hypothesis}


\section{Figures}

\begin{figure}[h]
	\begin{center}
		\includegraphics[width=\textwidth]{fig/nanostr-class/nanostr-valid.pdf}
	\end{center}
	\caption{Validation of classification assay on indepedent medulloblastoma cohorts.}
	\textbf{a-c}, Expression heatmaps of nanoString class-predicted medullboalstaoms of known subgroup status as published by Remke et al.\citeref{remke11} (a), Cho et al.\citeref{cho11} (b), and Kool et al.\citeref{kool08} (c). Samples are sorted according to subgroup predictions. Known expression subgroup affiliations and erroneously classified cases are marked above the heapmatp. \textbf{d}, \emph{Left}, Pie chart depicting the known subgroup distribution of medulloblastomas from the three independent cohorts analyzed in a-c ($n = 130$) and the subgroups predicted by nanoStringn profiling. Misclassified cases are marked within each slice according to the predicted subgroups. \emph{Right}, Pie chart of class prediction accuracy ($127/130$) from the validation set. Adapted from Northcott et al.\citeown{northcott12}
	\label{fig:nanostr-valid}
\end{figure}

\begin{figure}[h]
	\begin{center}
		\includegraphics[width=\textwidth]{fig/nanostr-class/nanostr-ffpe.pdf}
	\end{center}
	\caption{Classfication performance on formalin-fixed paraffin embedded (FFPE) archival samples.}
	\textbf{a}, Class prediction accuracy in relation to sample age of archival medulloblastomas stored as FFPE material ($n = 84$). Samples obtained with the past 8 years exhibit accuracies of $\geq 95\%$, as demarcated by the red vertical line. \textbf{b}, Heatmap of nanoString data showing class predictions for FFPE cases of $\leq 8$ years confidently predicted by assay ($n = 28$). Samples are sorted according to subgroup prediction. All cases satisfying prediction probability threshold were assigned to the correct subgroup ($28/28$). Adapted from Northcott et al.\citeown{northcott12}
	\label{fig:nanostr-ffpe}
\end{figure}

\clearpage

\begin{SCfigure}
	\centering
	\includegraphics[width=0.6\textwidth]{fig/magic-cn/cn-heatmap.png}
	\caption{Genome-wide copy-number profile of medulloblastoma subgroups.
	Copy-number profiling was performed on 1087 non-overlapping primary medulloblastomas. Shown is a copy number heatmap for 827 cases classified according to medulloblastoma subgroup based on matched gene expression data.  Amplifications are shown in red and deletions in blue.}
	\label{fig:cn-heatmap}
\end{SCfigure}

\begin{SCfigure}
	\centering
	\includegraphics[width=0.6\textwidth]{fig/magic-cn/broad-events.pdf}
	\caption{Frequency and significance of broad cytogeneetic events across medulloblastoma subgrouops.
	Gains are plotted in red and deletions in blue, shaded according to significance ($q < 0.1$, binomial test).}
	\label{fig:broad-events}
\end{SCfigure} 


\clearpage

\begin{SCfigure}
	\centering
	\includegraphics[width=0.5\textwidth]{fig/magic-cn/subgroup-specificity.pdf}
	\caption{Significant regions of focal SCNA identified by GISTIC2 in pan-cohort or subgroup-stratified analyses.
	A total of 62 significant regions were identified when the cohort was analyzed as a single group, whereas 110 significant regions were captured when the cohort was analyzed according to subgroup. The number of significant subgroup-enriched regions identified more than doubled (73 vs. 30) when the subgroups were analyzed independently.}
	\label{fig:subgroup-specificity}
\end{SCfigure}

\begin{SCfigure}
	\centering
	\includegraphics[width=0.5\textwidth]{fig/magic-cn/high-level-amps.pdf}
	\caption{Recurrent high-level amplifications in medulloblastoma.
	Frequency of genes amplified (segmented copy-number $\geq 5$) in at least two samples are shown with the distribution of the event across subgroups. The number of genes mapping to the peak region as defined by GISTIC2 (where applicable) are listed in parentheses after the candidate driver gene.}
	\label{fig:high-level-amps}
\end{SCfigure}

\begin{SCfigure}
	\centering
	\includegraphics[width=0.5\textwidth]{fig/magic-cn/homo-del.pdf}
	\caption{Recurrent homozygous deletions in medulloblastoma. Frequency of genes targeted by homozygous deletion (segmented copy-number $\leq 0.7$) in at least two samples are shown.}
	\label{fig:homo-del}
\end{SCfigure}

\begin{SCfigure}
	\centering
	\includegraphics[width=0.5\textwidth]{fig/magic-cn/nanostr-verification.pdf}
	\caption{Verification of focal SCNAs by nanoString.
	Genes inferred to be focally amplified by SNP6 were interrogated using a custom nanoString CodeSet across a set of 192 medullublastomas selected from our cohort. Bar graph shows the number of samples for which each gene is verified (red) or not (black). A verification rate of 90.9\% was achieved.}
	\label{fig:nanostr-verification}
\end{SCfigure}

\clearpage

\begin{figure}[h]
	\begin{center}
		\includegraphics[width=\textwidth]{fig/magic-cn/shh-gistic.pdf}
	\end{center}
	\caption{Landscape of SCNAs in SHH medulloblastoma.}
	GISTIC2 significance plot of amplifications (red) and deletions (blue) observed in the SHH subgroup is shown.
	Labeled cytobands indicate significant regions that satisfied multiple criteria to qualify as a potential somatic event. The number of genes mapping to each significant region are shown in parentheses next to the cytoband label.  Regions over-represented in SHH are highlighted in red.
	\label{fig:shh-gistic}
\end{figure}

\clearpage

\begin{SCfigure}
	\centering
	\includegraphics[width=0.4\textwidth]{fig/magic-cn/shh-signature.pdf}
	\caption{Significant overlap between SHH signature genes and genes targeted by focal SCNA in SHH.
		Bar graphs show the overlap significance based on permutation tests between SHH signature genes reported in the Cho (left) or Northcott (right) studies and genes mapping to focal SCNAs in our dataset. Dashed line indicates significance threshold ($alpha = 0.05$).}
	\label{fig:shh-signature}
\end{SCfigure}

\begin{SCfigure}
	\centering
	\includegraphics[width=0.5\textwidth]{fig/magic-cn/shh-amps-igv.pdf}
	\caption{Recurrent high-level amplifications of \emph{PPMID} and co-amplification of \emph{MDM4} and \emph{PIK3C2B} in SHH medulloblastom.
	Segmented copy-number tracks are shown for the amplified loci (17q23 and 1q23).}
	\label{fig:shh-amps-igv}
\end{SCfigure}

\begin{SCfigure}
	\centering
	\includegraphics[width=0.4\textwidth]{fig/magic-cn/mdm4-fish.jpg}
	\caption{Validation of \emph{MDM4} amplification in medulloblastoma.
	Interphase fluorescence in situ hybridization (FISH) of the \emph{MDM4} locus confirmed amplification in 8.2\% (12/146) of external cases present on a medulloblastoma tissue microarray (work by Andrey Korshunov).}
	\label{fig:mdm4-fish}
\end{SCfigure}

\clearpage

\begin{figure}[h]
	\begin{center}
		\includegraphics[width=\textwidth]{fig/magic-cn/shh-me.pdf}
	\end{center}
	\caption{Mutually exclusivity and clinical significance of focal SCNAs in SHH medulloblatoms.}
	Mutual exclusivity analysis of focal SCNAs in SHH medulloblastoma reveals that the most prevalent events in this subgroup do not co-occur in the same sample and collectively contribute to 23\% of cases, suggesting functional redundancy of SCNAs. Notwithstandingly, the mutually exclusive events exhibit distinct clinical outcome. Patients exhibiting either \emph{MYCN} or \emph{GLI2} amplification have poor clinical outcomes, while those exhibiting focal \emph{PTCH1} deletion have improved overall survival.
	\label{fig:shh-me}
\end{figure}

\begin{figure}[h]
	\begin{center}
		\includegraphics[width=\textwidth]{fig/magic-cn/shh-pathways.pdf}
	\end{center}
	\caption{Core pathways genetically targeted in SHH medulloblastoma.}
	Summary of SCNAs affecting components of Shh signaling, TP53 signaling, and RTK/PI3K signaling are depicted. Colours reflect the frequency by which the respective genes are targeted by focal or broad events in SHH medulloblastomas (red for amplification, blue for deletion). Significance values indicate the prevalence with which each pathway is targeted in SHH vs. non-SHH cases (Fisher's exact test).
	\label{fig:shh-pathways}
\end{figure}

\clearpage

\begin{figure}[h]
	\begin{center}
		\includegraphics[width=\textwidth]{fig/magic-cn/group3-gistic.pdf}
	\end{center}
	\caption{Landscape of SCNAs in Group 3 medulloblastoma.}
	GISTIC2 significance plot of recurrent amplifications and deletions in Group 3 is shown. Events highlighted in yellow are enriched in Group 3.
	\label{fig:group3-gistic}
\end{figure}

\clearpage

\begin{figure}[h]
	\begin{center}
		\includegraphics{fig/magic-cn/group3-me.pdf}
	\end{center}
	\caption{Mutual exclusivity of \emph{MYC}, \emph{OTX2}, and \emph{DDX31} aberrations in Group 3.}
	Analysis of focal SCNAs reveals that \emph{MYC} amplification and \emph{OTX2} amplifcation are completely mutually exclusivity, and these events are prognostic significant: \emph{MYC} amplified cases show poorer overall survival.
	\label{fig:group3-me}
\end{figure}

\begin{SCfigure}
	\centering
	\includegraphics[width=0.45\textwidth]{fig/magic-cn/group3-amps-igv.pdf}
	\caption{Recurrent amplifications target receptors of the TGF$\beta$ superfamily in Group 3.
		Segmented copy-number tracks of Group 3 medulloblastomas show recurrent high-level amplifications affecting \emph{ACVR2A} (2q22), \emph{ACVR2B} (3p22), and \emph{TGFBR1} (9q22).}
	\label{fig:group3-amps-igv}
\end{SCfigure}

\begin{SCfigure}[3.0]
	\centering
	\includegraphics[width=0.2\textwidth]{fig/magic-cn/acvr2b-fish.jpg}
	\caption{Validation of \emph{ACVR2B} amplification.
	FISH of the \emph{ACVR2B} locus confirmed presence of amplification in an external cohort of medulloblastomas on a TMA.}
	\label{fig:acvr2b-fish}
\end{SCfigure}

\begin{SCfigure}
	\centering
	\includegraphics[width=0.5\textwidth]{fig/magic-cn/group3-pathways.pdf}
	\caption{TGF$\beta$ signalling is recurrently disrupted by SCNAs in Group 3.
SCNAs affecting the TGF$\beta$ pathway comprise 20.2\% of Group 3 cases and are significantly enriched in Group 3 compared to non-Group 3 cases (Fisher’s exact test).}
	\label{fig:group3-pathways}
\end{SCfigure}

\clearpage

\begin{figure}[h]
	\begin{center}
		\includegraphics[width=\textwidth]{fig/magic-cn/group4-gistic.pdf}
	\end{center}
	\caption{Landscape of SCNAs in Group 4 medulloblastoma.}
	GISTIC2 significance plot of amplification and deletion regions in Group 4 is shown. Events highlighted in green are enriched in Group 4.
	\label{fig:group4-gistic}
\end{figure}

\clearpage

\begin{SCfigure}[2.0]
	\centering
	\includegraphics[width=0.3\textwidth]{fig/magic-cn/group4-dels-igv.pdf}
	\caption{NF-$\kappa$B pathway is recurrently targeted in Group 4.
		Recurrent focal deletions disrupt \emph{NFKBIA} and \emph{USP4}, negative regulators of the NF-$\kappa$B pathway, in Group 4 medulloblastoma.}
	\label{fig:group4-del-igv}
\end{SCfigure}

\begin{SCfigure}
	\centering
	\includegraphics[width=0.5\textwidth]{fig/magic-cn/group3-vs-group4.pdf}
	\caption{Aberrations disrupt distinct pathways in Group 3 and Group 4 medulloblastomas.
	Enrichment plot of gene sets disrupted by SCNAs in Group 3 vs. Group 4 medulloblastomas.}
	\label{fig:group3-vs-group4}
\end{SCfigure}

\clearpage

\begin{figure}[h]
	\begin{center}
		\includegraphics[width=\textwidth]{fig/magic-cn/sncaip-gain.png}
	\end{center}
	\caption{Focal gains recurrently target \emph{SNCAIP} in Group 4.}
	\emph{Left}, Segmented copy-number tracks of 317 Group 4 tumours shows a highly recurrent region of focal gain at 5q23.2 targeting a single gene: \emph{SNCAIP}, observed in 10.4\% of Group 4 cases (33/137). Whole-genome sequencing confirms that \emph{SNCAIP} is tandemnly duplicated\citeown{shih12}. \emph{Right}, Analysis of matched-germline confirms that \emph{SNCAIP} duplication is somatic.
	\label{fig:sncaip-gain}
\end{figure}

%Theme. Tandem duplication of SNCAIP defines a novel subtype of Group 4 medulloblastoma.
%Figure. Schematic representation of SNCAIP tandem duplication.  SNP6 copy number profile is shown for a sample with inferred SNCAIP gain and the net result as confirmed by WGS is output below the plot.

\begin{figure}[h]
	\begin{center}
		\includegraphics[width=\textwidth]{fig/magic-cn/sncaip-expr.pdf}
	\end{center}
	\caption{\emph{SNCAIP} is a Group 4 signature gene.}
	\emph{Right}, Box-plot depicting SNCAIP significant upregulation in Group 4 (Mann-Whitney test), as determined by expression analysis of a previously published cohort of 103 primary medulloblastoma \citeref{northcott11a}. SNCAIP ranks among the top 1\% of most highly expressed genes in Group 4 medulloblastoma (rank 39 out of 16758).
	\emph{Left}, Validation of SNCAIP as a Group 4 signature gene across five published medulloblastoma expression datasets: Thompson \citeref{thompson06}, Kool \citeref{kool08}, Fattet, Cho \citeref{cho11}, and Remke \citeref{remke11}. Expression datasets total 396 cases on four different array platforms. In all datasets, SNCAIP exhibited highest expression in Group 4.
	\label{fig:sncaip-expr}
\end{figure}

\begin{figure}[h]
	\begin{center}
		\includegraphics[width=\textwidth]{fig/magic-cn/group4-alpha-beta.pdf}
	\end{center}
	\caption{\emph{SNCAIP} duplication is restricted to one subtype of Group 4.}
	\textsf{a}, Non-negative matrix factorization (NMF) consensus clustering performed on expression profiles of Group 4 cases ($n = 188$) reveal two transcriptionally distinct subtypes of Group 4, designated $4\alpha$ and $4\beta$. SNCAIP duplicated is significantly enriched in the Group $4\alpha$ subtype (Fisher's exact test).
	\textsf{b}, SNCAIP expression is significantly elevated in Group $4\alpha$ compared to $4\beta$ (Mann-Whitney test).
	\textsf{c}, SNCAIP expression is copy number-driven in Group $4\alpha$. Group $4\alpha$ cases were stratified by \emph{SNCAIP} copy-number status. Samples harbouring SNCAIP duplication exhibit a significant \~1.5-fold increase in SNCAIP expression (Mann-Whitney test).
	\label{fig:group4-alpha-beta}
\end{figure}

\clearpage

% Figure. Genomic heterogeneity of Group 4 medulloblastoma.  GISTIC2 significance plots for Group 4α (i) and 4β (j) demonstrate disparate patterns of SCNA in the respective subtypes, with SNCAIP gain confined to 4α and gains of the prominent oncogenes MYCN and CDK6 restricted to 4β.

% Theme. Identification and verification of frequent PVT1-MYC fusion genes in Group 3 medulloblastoma.

\begin{figure}[h]
	\begin{center}
		\includegraphics[width=\textwidth]{fig/magic-cn/chromothr_myc_ai.pdf}
	\end{center}
	\caption{A multitude of amplicons disrupt the \emph{MYC}/\emph{PVT1} locus.}
	Copy-number plot of 8q24.21 is shown for a representative sample. Dots represent raw copy-number estimates and lines denote copy-number segments and state (red: gain, blue: loss). 71.4\% of \emph{MYC}-amplified (20/28) cases exhibit (partial) co-amplification of adjacent non-coding \emph{PVT1} gene and miR-1204. PVT1-MYC fusion transcripts were detected by RNA-seq, qRT-PCR, and Sanger sequencing in 8/20 samples \citeown{shih12}.
	\label{fig:chromothr_myc}
\end{figure}

\begin{figure}[h]
	\begin{center}
		\includegraphics[width=0.8\textwidth]{fig/magic-cn/chromothr_myc_wgs.pdf}
	\end{center}
	\caption{Chromothripsis disrupts the \emph{MYC}/\emph{PVT1} locus.}
	Whole-genome sequencing confirms a complex pattern of rearrangements on 8q24 in a representative sample, reminescent of chromothripsis (chromosome shattering).
	Plot shows chromosome 8 copy-number esimates (derived from read depth ratio of tumour vs. matched germline).
	Complex rearrangements are observed in concordance with a multitude of amplicons in the 8q24 region.
	\label{fig:chromothr_myc_wgs}
\end{figure}

\begin{figure}[h]
	\begin{center}
		\includegraphics[width=0.6\textwidth]{fig/magic-cn/chromothr-enrich.pdf}
	\end{center}
	\caption{Chromothripsis frequently disrupt chromosome 8 in Group 3 medulloblastoma.}
	Quantification of inferred chromothripsis across medulloblastoma subgroups reveal a significant enrichment of chromothripsis on chromosome 8 in Group 3 ($q = 0.0004$, Fisher's exact test), as compared to the entire cohort. Samples exhibiting at least 10 copy-number state changes on a single chromosome were inferred to have undergone chromothripsis.
	\label{fig:chromothr-enrich}
\end{figure}

\begin{figure}[h]
	\begin{center}
		\includegraphics{fig/magic-cn/myc-fusion_mir-expr.pdf}
	\end{center}
	\caption{miR-1204 is upregulated in Group 3 cases harbouring PVT1-MYC fusions.}
	Quantitative RT-PCR analysis of microRNAs hosted by PVT1 confirms upregulation of miR-1204 in Group 3 cases harbouring PVT1-MYC fusions (Mann-Whitney test). Samples were categorized as MYC-balanced/fusion(-) ($n = 4$), MYC-amplified/fusion(-) ($n = 6$), or MYC-amplified/fusion(+) ($n = 8$). Expression of microRNAs in MYC-balanced/fusion(-) samples served as baselines for determining relative expressions of the respective microRNAs in the other groups. Experiments were performed by John Peacock.
	\label{fig:myc-fusion_mir-expr}
\end{figure}

% Theme. Functional synergy between miR-1204 and MYC secondary to PVT1-MYC fusion in Group 3 medulloblastoma.

% Figure. Knockdown of miR-1204 attenuates the proliferative capacity of medulloblastoma cells harbouring PVT1-MYC fusion.  MED8A (PVT1-MYC fusion(+); b) or ONS76 (PVT1-MYC fusion(-); c) medulloblastoma cells were transfected with antagomiRs targeting miR-1204, siRNA against MYC, or a negative control and assessed for proliferation over 72 hours using the MTS assay.  MiR-1204 inhibition had a profound effect on the growth of MED8A cells but no effect on ONS76.

% Figure. Schematic representation illustrating the typical structure of the PVT1-MYC fusion observed in Group 3 medulloblastoma and it’s predicted functional significance.  

\clearpage

\begin{multicols}{2}
\small
\bibliographystyleref{nature}
\bibliographyref{mb}
\end{multicols}

\clearpage

\bibliographystyleown{nature}
\bibliographyown{shih-phd}


\end{document}

